### PR TITLE
Ros2 snowplow

### PR DESCRIPTION
**Pull request by girvenavery2022**
This pull request adds a package that can be launch on the physical robot. 
inside of the package their is a config directory, launch directory, and a urdf directory

### CODE REVIEW

File: mammoth.repos [hunk 1]
```diff
@@ -18,8 +18,11 @@ repositories:
...
     type: git
     url: https://github.com/iscumd/isc_serial.git
-    version: ros2   
   serial_utils_ros2:
     type: git
     url: https://github.com/iscumd/isc_serial_utils.git
-    version: ros2   
\ No newline at end of file
+    version: ros2     
+  ros2_ouster:
```
Comment (dcutting133) [inline mammoth.repos:25]: this should be an apt package, add it to cmakelists and package.xml and install using apt

File: mammoth_snowplow/src/mammoth_snowplow.cpp [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#include <cstdio>
+
+int main(int argc, char ** argv)
+{
+  (void) argc;
+  (void) argv;
+
+  printf("hello world mammoth_snowplow package\n");
+  return 0;
+}
```
Comment (dcutting133) [inline mammoth_snowplow/src/mammoth_snowplow.cpp:10]: is there any plan for this file?

File: deps.repos [hunk 3]
```diff
@@ -14,8 +14,11 @@ repositories:
...
     type: git
     url: https://github.com/iscumd/isc_serial.git
-    version: ros2   
   serial_utils_ros2:
     type: git
     url: https://github.com/iscumd/isc_serial_utils.git
-    version: ros2   
\ No newline at end of file
+    version: ros2     
+  ros2_ouster:
```
Comment (dcutting133) [inline deps.repos:21]: "

File: mammoth_snowplow/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+
+add_executable(mammoth_snowplow src/mammoth_snowplow.cpp)
+target_include_directories(mammoth_snowplow PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+
+install(
+  DIRECTORY launch config urdf
```
Comment (dcutting133) [inline mammoth_snowplow/CMakeLists.txt:30]: imo urdf should be in mammoth description package and pulled in as a dependency just as mammoth_gazebo does

**Review by dcutting133 — COMMENTED** [review]
Please rename all files and directories in snake_case.

File: mammoth_snowplow/src/mammoth_snowplow.cpp (hunk 2 above)
Comment (girvenavery2022) [reply mammoth_snowplow/src/mammoth_snowplow.cpp:10]: nope lol. i was in the process of fixing it then had to eat dinner

File: mammoth_snowplow/CMakeLists.txt (hunk 4 above)
Comment (girvenavery2022) [reply mammoth_snowplow/CMakeLists.txt:26]: fixed:)

File: mammoth.repos (hunk 1 above)
Comment (girvenavery2022) [reply mammoth.repos:25]: fixed

File: deps.repos (hunk 3 above)
Comment (girvenavery2022) [reply deps.repos:21]: fixed

**Pull request merged**